src/components/MultiSelect/MultiSelect.tsx

```diff
@@ -0,0 +1,104 @@
+import React from "react";
+import MultiSelectListBox from "./MultiSelectListbox";
+import MultiSelectDialog from "./MultiSelectDialog";
+import { MultiSelectItem, SelectedItems } from "./MultiSelectTypes";
+
+interface MultiSelectCommonProps {
+  /** The id of the multiSelect. */
+  id: string;
+  /** The label of the multiSelect. */
+  label: string;
+  /** The variant of the multiSelect. */
+  variant: "listbox" | "dialog";
+  /** The items to be rendered in the multiselect as options. */
+  items: MultiSelectItem[];
+  /** The selected items state (items that were checked by user). */
+  selectedItems: SelectedItems;
+  /** Value used to set the width for the MultiSelect component. */
+  width?: "default" | "fitContent" | "full";
+  /** Set the default open or closed state of the multiselect. */
+  defaultIsOpen?: boolean;
```

Comment:
This should be renamed to `isDefaultOpen` to be consistent with the `Accordion`'s `isDefaultOpen` prop.

Reply:
fixed.